### ISSUE TITLE
Full-screen dialer with dial pad tones?

### ISSUE THREAD

**Issue by Skaalgard** [SimpleMobileTools/Simple-Dialer#117]
Would it be possible for us to get a fullscreen dialer with dial pad tones?

**Comment by ltGuillaume** [comment]
Yes, please! It would be nice to have that as a new tab and to be able to choose it from the setting "Tab to open at app start".

**Comment by Skaalgard** [comment]
I agree. If I open the _dialer_, I generally want to _dial_ a number. If I'm specifically looking to call a contact, I usually open my contacts.

**Comment by ltGuillaume** [comment]
Yep, so true. As for the dialer part, though, there's quite some room for improvement regarding the T9 contact search algorithm in order for the app to become actually useful imho (https://github.com/SimpleMobileTools/Simple-Dialer/issues/91).

**Comment by seniorm0ment** [comment]
I agree, the dialer feels a bit small.
It would be nice to see the numbers spaced out slightly more, and maybe have some clear area definition to them to avoid misclicks, as well as a fullscreen dialer with tones.
